assets/section-password.css

```diff
@@ -107,7 +107,6 @@ body {
 }
 
 .password-field.field {
-  display: block;
   flex: 1 20rem;
 }
```

Comment:
This should be fixed on Tyler's PR, but it would be good to check: 

https://screenshot.click/18-41-ykvw3-xp5wy.png

Reply:
Yep! I didn't change the contact-form in my PR. The reason I noticed the issue is because I added the section to the password page. 👍🏻